TypeScript.YAML-tmLanguage

```diff
@@ -1375,7 +1375,7 @@ repository:
       match: '{{startOfIdentifier}}(as)\s+(const)(?=\s*([,}]|$))'
       captures:
         '1': { name: keyword.control.as.ts }
-        '2': { name: storage.modifier.ts }
+        '2': { name: storage.modifier.const.type.ts }
```

Comment:
please do not change scope for modifier `const`. Instead add `name` as `cast.expr.ts`